.github/workflows/publish.yml

```diff
@@ -42,5 +61,63 @@ jobs:
           publish: pnpm changeset:publish
           createGithubReleases: true
 
+  publish_docs:
+    name: Publish Docs
+    needs: publish
+    runs-on: ubuntu-latest
+    # For whatever reason, yaml does not like the full "meta(changelog): Update package versions" string
+    # So we check this in two parts
+    if: |
+      contains(github.event.head_commit.message, 'meta(changelog)') 
+      && contains(github.event.head_commit.message, 'Update package versions')
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+
+      - name: Import GPG key
+        id: import-gpg
+        uses: crazy-max/ghaction-import-gpg@v6
+        with:
+          gpg_private_key: ${{ secrets.RELEASER_GPG_PRIVATE_KEY }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+          git_config_global: true
+
+      - name: Setup Git
+        run: |
+          echo "machine github.com login x password ${{ secrets.GITHUB_TOKEN }}" > ~/.netrc
+          git config --global url."https://github.com/".insteadOf "git://github.com/"
+          git config --global advice.detachedHead false
+
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: "package.json"
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
+          run_install: false
+
+      - name: Get pnpm store directory
+        shell: bash
+        run: |
+          echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+
+      - name: Cache node_modules
+        id: cache-node-modules
```

Comment:
I don't think this will also save down node_modules to the cache and I'm not sure I see that happening anywhere else in the file.

Reply:
Alrighty removed extra uses of cache throughout all of the CI